packages/opds-web-client/src/components/__tests__/DownloadButton-test.tsx

```diff
@@ -111,34 +116,31 @@ describe("DownloadButton", () => {
   });
 
   it("fulfills OPDS-based indirect links", async () => {
-    let streamingType =
-      "text/html;profile=http://librarysimplified.org/terms/profiles/streaming-media";
+    const streamingLink: FulfillmentLink = {
+      ...fulfillmentLink,
+      type: "application/atom+xml;type=entry;profile=opds-catalog",
+      indirectType:
+        "text/html;profile=http://librarysimplified.org/terms/profiles/streaming-media"
+    };
     let downloadButton = providerWrapper(
-      <DownloadButton
-        style={style}
-        url="download url"
-        mimeType="application/atom+xml;type=entry;profile=opds-catalog"
-        indirectType={streamingType}
-        title="title"
-      />
+      <DownloadButton style={style} title="title" link={streamingLink} />
     );
     wrapper = mount(downloadButton);
     let button = wrapper.find("button");
     await button.simulate("click");
     expect(indirectFulfill.callCount).to.equal(1);
     expect(indirectFulfill.args[0][0]).to.equal("download url");
-    expect(indirectFulfill.args[0][1]).to.equal(streamingType);
+    expect(indirectFulfill.args[0][1]).to.equal(streamingLink.indirectType);
   });
 
   it("fulfills ACSM-based indirect links", async () => {
+    const ascmLink: FulfillmentLink = {
+      url: "download url",
+      type: "application/vnd.adobe.adept+xml",
```

Comment:
The original value for this was `"vnd.adobe/adept+xml"` and it should stay the same. The mimeType function can also be tested separately for this.

Reply:
Okay, makes sense. The `MediaLink` type doesn't have that type in the enum: https://github.com/NYPL-Simplified/opds-web-client/blob/b66c88f2df2dc2e564b813137b2526da7ad53414/packages/opds-web-client/src/interfaces.ts#L3

I will add it, but that then requires that I add an entry for it in the `typeMap` found here: 
https://github.com/NYPL-Simplified/opds-web-client/blob/b66c88f2df2dc2e564b813137b2526da7ad53414/packages/opds-web-client/src/utils/file.ts#L10

Do you know what the extension and format name should be? For now I will use `.acsm` and `ACSM` for now.

Reply:
Not sure but `.acsm` and `ACSM` is fine for now.